designs/administrator/4.6/guided-tours/index.md

```diff
@@ -0,0 +1,102 @@
+---
+parent: Administrator
+version: 4.6
+---
+
+# Guided tours
+
+In 4.6, guided tours will be added to the console. This documentation will outline how tours are designed and should behave. The content of the tour will not be detailed here. Content copy can be found [here](https://docs.google.com/document/d/1qEBDNZYaKF4TDogJLYUqX-8xHCmPCKdaHfKLxMlnZQs/edit?usp=sharing), where it will be reviewed and updated as needed.
+
+## Accessing tours
+Tours can be accessed on both the Administrator and Developer perspectives by going to the **Help menu** and selecting **Guided Tours**. Each of the perspectives will also surface links to the tours in other places around the console. For more detail around how these links will be surfaced in the Developer perspective, view the Developer side documentation.
+
+In the Administrator perspective, a badge will appear on the Overview page informing users that guided tours are available. When clicked, the badge will redirect users to the Guided Tours page.
+
+![1](img/admin1.png)
+
+The Guided Tours page will use a catalog-like design with each tour being shown on a tile. Tiles will provide a brief description of the tour, a time estimate for completing the tour, as well as prerequisites for completing the tour, if any.
+
+![2](img/admin2.png)
+
+If a user clicks on a tour tile, the tour side panel will open. The introductory panel will give users more information and allows the user to start the tour using the **Start tour** button.
+
+![3](img/tour1-screen0.png)
+
+## Tour permissions
+For 4.6, tour permissions will be as follows:
+* If a user can install Operators, show the first Serverless tour (Explore Serverless) which walks users through installing the Serverless Operator.
+* If a user cannot install Operators, do not show the first Serverless tour. Instead show an empty state. The empty state can inform users that tours will be added in future.
+* If an Operator has been installed, show the second Serverless tour (Serverless Applications) which walks users through deploying a Serverless application.
+* Any tour that has been completed should remain on the Guided Tours page with the option to **Review the tour** or **Restart the tour**.
```

Comment:
The "Restart the tour" option mentioned here seems to be missing from `tour2-complete.png` below.

Reply:
Ah good catch! I removed it from the images and will do so here too. When I thought more about it, I wasn't sure how these two buttons would differ and since we can't delete anything you've done through the tour before, "Restart" seems a little misleading.